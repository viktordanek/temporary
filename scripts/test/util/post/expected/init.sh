TARGET=${e55dd2c8db9b224d0d6207c430354f481ece26fbf458400726e7624bcc79fcb72de81bccc55a066ebfa569317862dec4b13ea6bb4b1e8b0300f1dc867e51503d} &&
  export INIT_ARGUMENTS=wrong5 &&
  TEMPLATE=$( ${SED} -e "s#\"#\\\"#g" ${TEMPLATE_FILE} ) &&
  ${JQ} -n --arg INIT_ARGUMENTS wrong6 "$( ${SED} -e "s#\"\(env(.*)\"\"#$1#" -e "s#\"#\\\"#g" ${TEMPLATE_FILE} )" > ${TARGET}