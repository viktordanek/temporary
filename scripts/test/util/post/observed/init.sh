TARGET=${e55dd2c8db9b224d0d6207c430354f481ece26fbf458400726e7624bcc79fcb72de81bccc55a066ebfa569317862dec4b13ea6bb4b1e8b0300f1dc867e51503d} &&
  ${FIND} ${@} -mindepth 1 -maxdepth 1 -type f | ${SORT} | while read FILE
  do
    BASE=$( ${BASENAME} ${FILE} ) &&
        ${ECHO} "- name: ${BASE}" >> ${TARGET} &&
        if [ -f ${FILE} ]
        then
          ${ECHO} "  type: file" >> ${TARGET}
        elif [ -L ${FILE} ]
        then
          ${ECHO} "  type: link" >> ${TARGET}
        else
          ${ECHO} "  type:  other" >> ${TARGET}
        fi
    done