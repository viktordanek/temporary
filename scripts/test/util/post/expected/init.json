[ { content : env(ARGUMENTS) , name : "arguments" , type : "file" } ]